motion/p5js/spiral-beams/index.md

```diff
@@ -3,7 +3,7 @@ layout: p5js
 author: Marius Claßen
 author-url: https://github.com/mariusclassen
 title: spiral beams
-tags: [P5.js, motion, Law of Regularity]
+tags: [P5.js, motion, law of regularity]
```

Comment:
@mariusclassen Unfortunately (this is a design flaw done by me) are the tags for the Laws Title Case. Sorry for that. This is something I will have to fix.